var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -326,11 +326,11 @@ def cmake_args(self):
 
         return args
 
-    phases = ("cmake", "build", "install", "pip_install_nodeps")
-    build_targets = ["all"]
-    with when("+python"):
-        build_targets += ["pip_wheel"]
+    phases = ["cmake", "build", "install"]
+    #with when("+python"):
+    #    phases += ["pip_install_nodeps"]
```

Comment:
The moment I add this part, it adds the extra phase even if `~python` is spec'd

Reply:
With this block, both
- `spack info --phases "warpx ~python"` and
- `spack info --phases "warpx +python"`

show four phases, including the `"pip_install_nodeps"`. I think that is a bug.

Reply:
@adamjstewart do you know why?